src/org/scratchOrgInfoApi.ts

```diff
@@ -295,6 +295,10 @@ export const requestScratchOrgCreation = async (
 
   const scratchOrgInfo = mapKeys(scratchOrgRequest, upperFirst, true);
 
+  if (typeof scratchOrgInfo.Username === 'string') {
```

Comment:
just checking I understand this correctly:
`Username` is a string: https://github.com/forcedotcom/sfdx-core/blob/14f4d72dff0d383d709dbf678b2e42976f485467/src/org/scratchOrgTypes.ts#L38
but could be undefined at runtime so you check the type to be safe?